test/core/handshake/client_ssl.cc

```diff
@@ -161,7 +161,9 @@ static void server_thread(void* arg) {
   // Set the cipher list to match the one expressed in
   // src/core/tsi/ssl_transport_security.c.
   const char* cipher_list =
-      "ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-"
+      "TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_"
+      "SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-"
+      "AES256-"
       "SHA384:ECDHE-RSA-AES256-GCM-SHA384";
```

Comment:
I don't believe adding TLS 1.3 ciphers actually does anything here. In BoringSSL, we do not allow configuring TLS 1.3 ciphers at all and just have hardcoded preferences within the library. (This helps us ensure we get the AES vs ChaCha20 negotiation correct. We've found that, most of the time, external folks configuring it for TLS 1.2 got it wrong.) In OpenSSL, it is possible to configure, but only via a different API. If you only call this API, it'll use its defaults (which are also sensible).

Ditto for the server change below.

Reply:
Thanks, I didn't realize! I've removed the TLS 1.3 ciphersuites from this list.